src/main/java/com/github/romankh3/image/comparison/model/Rectangle.java

```diff
@@ -134,7 +134,7 @@ public Integer size() {
      * @return rectangle width.
      */
     public int getWidth() {
-        return maxPoint.getX() - minPoint.getX();
+        return maxPoint.getX() - minPoint.getX() + 1; // min and max point are included, so real width is +1px
```

Comment:
It is not good idea, because wight, height size will be incorrect.
Example we have difference in next 3 pixels:
5, 3; 6, 3; 6, 4
In the case each will be: 5, 3 - 6, 4. With you proposed code width is 1px, but real width is 2, height is 1px, but real width is 2 px, tech size in your case is 1, but we have rect with size 4 which contains 3 different pixels

Reply:
In changes I thing you saw that I updated MinimalRectangleSize in one test, because additional added point was 4x4. And this should be done in other code which try to set MinimalRectangleSize. So my point it is beter to leave this fix and all clients of new version should update there code.